types/react-speech-recognition/index.d.ts

```diff
@@ -23,6 +23,7 @@ interface SpeechRecognition {
     stopListening(): void;
     abortListening(): void;
     browserSupportsSpeechRecognition(): boolean;
+    applyPolyfill(speechRecognitionPolyfill: any): void;
```

Comment:
As per [react-speech-recognition's docs](https://github.com/JamesBrill/react-speech-recognition/blob/master/docs/POLYFILLS.md) we should be able to narrow this any type down to a implementation of the [W3C SpeechRegcognition spec](https://wicg.github.io/speech-api/#speechreco-section)